.github/workflows/build-and-deploy-on-pypi.yml

```diff
@@ -27,21 +27,46 @@ jobs:
           pep517
           --user
       - name: Build a binary wheel and a source tarball
+        id: build_tarball
         run: >-
           python -m
           pep517.build
           --source
           --binary
           --out-dir dist/
           .
+        # store version and SHA
+        run: |
+          version=$(sed '3q;d'  ESMValCore.egg-info/PKG-INFO | cut  -d ":" -f 2)
+          sha=$(shasum -a 256 dist/esmvalcore-${version}.tar.gz | awk '{print $1}')
+          echo "::set-output name=tarball-sha::${sha}"
+          echo "::set-output name=pypi-version::${version}"
       #- name: Publish distribution 📦 to Test PyPI
       #  uses: pypa/gh-action-pypi-publish@master
       #  with:
       #    password: ${{ secrets.test_pypi_password }}
       #    repository_url: https://test.pypi.org/legacy/
-      - name: Publish distribution 📦 to PyPI
+      - name: Publish distribution 📦 to PyI
         if: startsWith(github.ref, 'refs/tags')
         uses: pypa/gh-action-pypi-publish@release/v1
+        id: pypi_published
         with:
           user: __token__
           password: ${{ secrets.pypi_password }}
+      - name: Make PR on conda forge feedstock
+        if: startsWith(github.ref, 'refs/tags')
+        if: steps.pypi_published.outputs.exit_code == 0
```

Comment:
Maybe it would be nice to use this feature instead?
https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-requiring-successful-dependent-jobs

Reply:
will do, cheers! - am a bit fed up of GA stuff for today, will come back tomorrow :grin: